iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py

```diff
@@ -0,0 +1,68 @@
+"""Helper classes showing interactive timeseries."""
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *
+from .viewer_base import BaseViewer
+
+
+class HistogramViewer(BaseViewer):
+    """A histogram viewer based on quad() Glyph.
+
+    All functionality in BaseViewer is present and there are a few
+    convenience functions for setting the tick format on the axes
+    easily.
+
+    Apart from the special arguments listed below, all other arguments
+    are passed through to BaseViewer.
+
+    Args:
+        left
+        x_decade_mantissa (list): The mantissa values to use for the x_axis.  If
+            not specified this defaults to ticks on 1, 2 and 5 mantiss values.
+        x_sciformat (bool): Whether to display the x tick values in 10^X format
+            or in normal decimal.  Defaults to scientific.
+        y_decade_mantissa (list): The mantissa values to use for the y_axis.  If
+            not specified this defaults to ticks on 1, 2 and 5 mantiss values.
+        y_sciformat (bool): Whether to display the y tick values in 10^X format
+            or in normal decimal.  Defaults to scientific.
+    """
+
+    def __init__(self, **kwargs):
+
```

Comment:
This extra space isn't needed.

Reply:
Got it. Will remove these extra spaces and commit.